horse/models/user.py

```diff
@@ -0,0 +1,20 @@
+class User:
+    def __init__(
+            self, user_id, name, followed_users=None, liked_movies=None
+    ):
+        self.user_id = user_id
```

Comment:
Hm, ids are not used in your implementation, we cancould safely remove them for now